scripts/prepare/config.ts

```diff
@@ -1,3 +1,7 @@
+import {glob} from "glob";
+import {readFile} from "node:fs/promises";
+import type {PackageJson} from "type-fest";
```

Comment:
Not used. Also, Prettier complains about the code. Not sure why it didn't fail during the style testing...